apps/frontend/src/components/RequestBox/RequestBox.tsx

```diff
@@ -46,18 +55,25 @@ export default function RequestBox({ requests }) {
         Create request
       </button>
       <div className={styles.form}>
-        <SearchFilterForm
-          filterStatus={requestsType}
-          setFilterStatus={setRequestsType}
-          requestSearchValue={requestSearchValue}
-          setRequestSearchValue={setRequestSearchValue}
-        />
+        <Form className={styles.form}>
+          <Form.Group>
+            <Form.Control type="text" placeholder="Search requests by title" value={searchCriteria.searchTerm} 
+            onChange={event => searchRequests((event.target).value)}></Form.Control>
+          </Form.Group>
+          <div className={styles.inputGroup}>
+            <Form.Select size='sm' className={styles.selectBox} value={searchCriteria.status}
+              onChange={event => filterRequests((event.target).value)}>
+              <option value='OPEN'>Open Requests</option>
+              <option value='CLOSED'>Closed Requests</option>
+              <option value='ALL'>All Requests</option>
+            </Form.Select>
+          </div>
+        </Form>
       </div>
-      <RequestModal show={show} handleClose={handleClose} />
+      <CreateRequestModal show={show} handleClose={handleClose} />
       <div className={styles.container}>
-        {requestBoxes}
+        {requestBoxes.length !== 0 ? requestBoxes : 'Currently there are no requests that match your criteria!'}
```

Comment:
There's a styling bug here: the inputs overlap the section title and create request button when there are no results to display.

Reply:
Can you share a screenshot please?

Reply:
![image](https://github.com/neighborhood-app/app/assets/58592210/64b27ad6-4281-4aef-825f-8a5c5e3dcccd)

Reply:
I can't manage to reproduce the bug  
![Screenshot 2023-10-04 063425](https://github.com/neighborhood-app/app/assets/40914437/c457660e-37a2-452d-a985-da870bfaabe8)

Reply:
all right let's leave it for now since there will be more changes with the buttons and everything and I'll see if it persists in the future